ansible/roles/test/files/ptftests/py3/hash_test.py

```diff
@@ -105,6 +106,7 @@ def setUp(self):
         self.ipver = self.test_params.get('ipver', 'ipv4')
         self.is_active_active_dualtor = self.test_params.get("is_active_active_dualtor", False)
         self.topo_name = self.test_params.get('topo_name', '')
+        self.is_v6_topo = "isolated-v6" in self.topo_name
```

Comment:
Hi @wrideout-arista , we have a common utility for detecting ipv6 topo now. https://github.com/sonic-net/sonic-mgmt/blob/edfafd7a6fda9cffde20e8028be669ce3ceee9e3/tests/common/utilities.py#L993 Thanks